src/components/DatePicker/DatePicker.stories.mdx

```diff
@@ -41,3 +42,40 @@ Implementation of the simple DayPicker
     <DatePicker />
   </Story>
 </Preview>
+
+# DayPicker with disabled dates
```

Comment:
Probably you are about to hate me... and I am really, really sorry for that, but since the `disabledDates` property of the `DatePicker` component has many options that can be given as standalone properties or can be combined, I think it should be useful to add all the available options inside the **Docs** tab, along with a combination of options as an example, or even add them as a knob. 😞 (sorry)

Reply:
as this has many options but defined in the property type as Modifier so that the developer can easily know what to add. If i put it with knobs means that there is an interest for someone outside the FE team to play with the component.

How do you suggest to put it on knobs?

Reply:
I totally agree for the use outside FE on the Docs part!
But if you consider that the provided options are from the Datepicke library and someone inside the team might not know how to specify them, it would be useful to have them inside the code for reference!
Also, for the knob, I was thinking maybe a dropdown with some examples, so that a designer as well could play with it and have a more clear image on the capability of the library used.